app/models/job.rb

```diff
@@ -3,4 +3,7 @@ class Job < ActiveRecord::Base
   STATUSES = [:draft, :published, :archived]
   enumerize :state, in: STATUSES
   belongs_to :organization
+  belongs_to :published_by, foreign_key: :user_id, class_name: "User"
```

Comment:
It could totally be done the way you wrote: `belongs_to :user`. It would have the benefit of being terser.

However, I would invoke the argument of "code readability" as a justification here. I think the concept of a "publisher" describes the problem domain better than a "user". When attempting to find out who published (i.e. authored) the model (be it a job, an event or a news_item), I think

```ruby
Job.first.published_by
```

reads better than:

```ruby
Job.first.user
```

As an alternative, we could define that `belongs_to` relationship with `publisher` (or even `author`) instead of `published_by`.